core/base/src/TApplication.cxx

```diff
@@ -468,7 +468,46 @@ void TApplication::GetOptions(Int_t *argc, char **argv)
          } else {
             Warning("GetOptions", "-e must be followed by an expression.");
          }
+      } else if (!strcmp(argv[i], "--")) {
+         TObjString* macro = NULL;
```

Comment:
Can you use `nullptr` instead of `NULL`? (I know, not obvious when looking at surrounding code...)